.github/workflows/handle.yml

```diff
@@ -43,9 +43,14 @@ env:
   DEFAULT_DURATION: '2h'
 
 jobs:
-  handle:
+  yaml_lint:
     # Only handle the request when the PR is approved.
     if: '${{ github.event.review.state == ''approved'' }}'
+    uses: 'abcxyz/pkg/.github/workflows/yaml-lint.yml@main' # ratchet:exclude
+
+  handle:
+    # Only run when yaml_lint is successfully completed.
+    needs: 'yaml_lint'
```

Comment:
This creates a dependency - do we want to require lint to pass _before_ validation, or should they happen at the same time?

Reply:
I feel like the dependency is needed here since we want requestor to fix the lint issue before handling the request. For the validate.yml though, we probably can run them at the same time.

Reply:
Since we already yaml lint in the validate workflow, we don't need to rerun it to cause duplicated checks.

Reply:
In summary, the yaml lint is just a reference for request reviewer/approver. updating!